dashboards/MongoDB/MongoDB_Cluster_Summary.json

```diff
@@ -819,7 +1393,11 @@
       "type": "stat"
     },
     {
-      "description": "Show the data distribution between all the shards involved",
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PA58DA793C7250F1B"
```

Comment:
I believe we can use `prometheus` (etc.) as the uid since https://github.com/percona/pmm/pull/3926/files is merged.